.github/TRANSLATING.md

```diff
@@ -0,0 +1,33 @@
+# Translations (i18n)
+
+Thanks for your interest in helping to translate the documents! While we plan to support translations in the future, **we are not currently accepting them until Tauri 2.0 is a bit more stable**.
+
+Once translations are enabled you can check the [translation status page](https://beta.tauri.app/contribute/translate-status).
+
+To ensure the translation is consistent across multiple contributions, be sure to [communicate with others contributors in the `#docs` channel on Discord](https://discord.com/invite/tauri) and consult with reliable external resources such as [Microsoft terminology search](https://msit.powerbi.com/view?r=eyJrIjoiODJmYjU4Y2YtM2M0ZC00YzYxLWE1YTktNzFjYmYxNTAxNjQ0IiwidCI6IjcyZjk4OGJmLTg2ZjEtNDFhZi05MWFiLTJkN2NkMDExZGI0NyIsImMiOjV9) during translation.
+
+### Enabling a Language
+
+If you seek to contribute but find your language missing, please open an issue.
```

Comment:
Maybe we could add a checklist for maintainers here?

- Update https://github.com/tauri-apps/tauri-docs/blob/next/locales.json
- Update https://github.com/tauri-apps/tauri-docs/blob/next/.github/labeler.yml

Those are the two I know of off the top of my head, but might be more we catch when we enable a lang